modules/identities/scripts/list.ts

```diff
@@ -0,0 +1,23 @@
+import { ScriptContext } from "../module.gen.ts";
+import { listIdentities } from "../utils/db.ts";
+import { IdentityProviderInfo } from "../utils/types.ts";
+
+export interface Request {
+	userToken: string;
+}
+
+export interface Response {
+    identityProviders: IdentityProviderInfo[];
+}
+
+export async function run(
+	ctx: ScriptContext,
+	req: Request,
+): Promise<Response> {
+	await ctx.modules.rateLimit.throttlePublic({});
```

Comment:
is list meant to be public? currently it's private with a rate limit.

Reply:
AFACT, according to the `module.json`, it is public.